app/controllers/api/v2/courses_controller.rb

```diff
@@ -59,6 +52,7 @@ def publish
       def update
         update_enrichment
         update_sites
+        sync_with_search_and_compare if site_ids.present?
```

Comment:
Similar to the comment about `publish`, I don't think we should be relying on code in an existing action here.

Reply:
Same as above.